templates/ding-item-list_item-table.tpl.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @file
+ * Template file for single item list item.
+ */
+$alt = check_plain($title . ' ' . $author);
+?>
+<table class="ding-item-list-item">
+  <tbody>
+  <tr>
+    <td class="item-cover">
+      <a href="/ting/object/<?php print $faust; ?>">
```

Comment:
It is sent per email so it should be absolute and not relative.

Reply:
It is absolute link. https://github.com/easySuite/emailkanon/blob/development/emailkanon.module#L1377

Reply:
@guddo Agree with @NicolaeGrigoriu. We have a function _absolute_url() which is taking care of this.